File: infra/lambda/read_audio.py
import boto3
import logging

s3_client = boto3.client("s3")

def handler(event, context):
    bucket_name = event["Records"][0]["s3"]["bucket"]["name"]
    file_name = event["Records"][0]["s3"]["object"]["key"]
    resp = s3_client.get_object(Bucket=bucket_name,Key=file_name)
    logging.info((resp))
    data = resp['Body'].read()
    logging.info((data))
    return data

File: lambda/read_audio.js
const aws = require('aws-sdk');

const s3 = new aws.S3({ apiVersion: '2006-03-01' });

exports.handler = async (event, context) => {
    console.log('Received event:', JSON.stringify(event, null, 2));

    const bucket = event.Records[0].s3.bucket.name;
    const key = decodeURIComponent(event.Records[0].s3.object.key.replace(/\+/g, ' '));
    const params = {
        Bucket: bucket,
        Key: key,
    }; 
    try {
        const { ContentType, Body } = await s3.getObject(params).promise();
        console.log('BODY:', Body);
        console.log('CONTENT TYPE:', ContentType);
        return ContentType;
    } catch (err) {
        console.log(err);
        throw new Error(message);
    }
};
              